plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+/**
+ * Collects all data needed to create and configure an App Engine Standard Project.
+ */
+class AppEngineStandardProjectConfig {
+
+  private String appEngineProjectId;
```

Comment:
I wanted to duistinguish between the *Eclipse* project and the *App Engine* project.